include/RAJA/policy/cuda/atomic.hpp

```diff
@@ -229,8 +243,11 @@ RAJA_INLINE __device__ double cuda_atomicAdd<double>(double volatile *acc,
 {
   return ::atomicAdd((double *)acc, value);
 }
+#elif __CUDA_ARCH__ < 600 
+#warning CUDA D ATOMICADD NOT COMPILED, NEED TO SET nvcc -arch=sm_60 (__CUDA_ARCH__ >= 600)
```

Comment:
I'm not sure that we want this warning to always appear when compiling with sm_35.
It would be confusing because we do provide an implementation of cuda atomicAdd double in that case via the generic template version of the function.